Source/AMRInterpolator/InterpSource.hpp

```diff
@@ -19,7 +19,9 @@ class InterpSource
     virtual bool
     contains(const std::array<double, CH_SPACEDIM> &point) const = 0;
     virtual void
-    fillAllGhosts(const VariableType var_type = VariableType::evolution) = 0;
+    fillAllGhosts(const VariableType var_type = VariableType::evolution,
+                  const Interval &a_comps =
+                      Interval(0, std::numeric_limits<int>::max())) = 0;
```

Comment:
Would it be better to default the interval to the max of NUM_VARS and NUM_DIAGNOSTIC_VARS?

Reply:
Ok, ignore this as it happens in fillAllGhosts()